JustLearn.Services.Api/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
I am not sure if this file can go with date time commit or not.

Reply:
I think it was missed in the initialize commit. So it is fine to be added now.